packages/cli/tests/unit/rpc/explain/index/appmap-index.spec.ts

```diff
@@ -63,7 +63,7 @@ describe('readAppMapContent', () => {
     expect(content).toContain('sql');
     expect(content).toContain('database');
 
-    expect(content.split(' ')).toEqual([
+    expect(content.split(/\s+/)).toEqual([
```

Comment:
The content is no longer joined by spaces, so splitting on whitespace restores the previous behavior. This is functionally the same as how we're [splitting words in tokenize](https://github.com/getappmap/appmap-js/blob/355d931458dded962a186de86722c451e75c530d/packages/search/src/tokenize.ts#L59), even though the regex is different.